modules/core/src/v2/coinFactory.ts

```diff
@@ -210,3 +222,15 @@ for (const token of [...tokens.bitcoin.algo.tokens, ...tokens.testnet.algo.token
   const tokenConstructor = AlgoToken.createTokenConstructor(token);
   GlobalCoinFactory.registerCoinConstructor(token.type, tokenConstructor);
 }
+
+for (const token of [...tokens.bitcoin.avaxc.tokens]) {
+  const tokenConstructor = AvaxCToken.createTokenConstructor(token);
+  GlobalCoinFactory.registerCoinConstructor(token.type, tokenConstructor);
+  GlobalCoinFactory.registerCoinConstructor(token.tokenContractAddress, tokenConstructor);
+}
+
+for (const token of [...tokens.testnet.avaxc.tokens]) {
```

Comment:
duplicated code, you can refactor extract a function with common code

Reply:
it's not exactly duplicated.

Reply:
Yes, the list and the "accept duplicates" boolean parameter